src/taxonomy/import-tags/ImportTagsWizard.test.jsx

```diff
@@ -230,16 +229,15 @@ describe('<ImportTagsWizard />', () => {
     if (expectedResult === 'success') {
       // Toast message shown
       await waitFor(() => {
-        expect(mockSetToastMessage).toBeCalledWith(`"${sampleTaxonomy.name}" updated`);
+        expect(mockSetToastMessage).toHaveBeenCalledWith(`"${sampleTaxonomy.name}" updated`);
```

Comment:
Not related to the issue, but toBeCalledWith is `deprecated` (ref https://github.com/jestjs/jest/issues/13164)